build-tools/packages/version-tools/src/internalVersionScheme.ts

```diff
@@ -374,10 +376,10 @@ export function bumpInternalVersion(
  */
 export function getVersionRange(
 	version: semver.SemVer | string,
-	maxAutomaticBump: "minor" | "patch" | "~" | "^",
+	maxAutomaticBump: "minor" | "patch" | "~" | "^" | "legacyCompat",
+	defautRange?: number,
 ): string {
-	validateVersionScheme(version, true, undefined);
-
+	// validateVersionScheme(version, true, undefined);
```

Comment:
I think you want to keep this.

Reply:
this was commented for testing purposes. uncommented now.